napi_wrap crashes on Node 14, Parcel 2.0.0-nightly-722

# 🐛 bug report

A crash happens on Node 14, Parcel 2.0.0-nightly-722

Windows error:
https://github.com/aminya/solid-simple-table/pull/11/checks?check_run_id=2816163871#step:4:58
```js
#
# Fatal error in , line 0
# Check failed: result.second.
#
#
#
#FailureMessage Object: 000000486D0FE600
 1: 00007FF705373E0F napi_wrap+108911
 2: 00007FF7052A67DF std::basic_ostream<char,std::char_traits<char> >::operator<<+57423
 3: 00007FF705F00CB2 V8_Fatal+162
 4: 00007FF705990FED v8::internal::BackingStore::Reallocate+653
 5: 00007FF705BD7829 v8::ArrayBuffer::GetBackingStore+137
 6: 00007FF70533882A node::Buffer::Data+58
 7: 00007FFEC0FBA261 
 8: 00007FFEC0FB5B57 
 9: 00007FF705B9DDCF v8::internal::Builtins::builtin_handle+321711
10: 00007FF705B9D364 v8::internal::Builtins::builtin_handle+319044
11: 00007FF705B9D657 v8::internal::Builtins::builtin_handle+319799
12: 00007FF705B9D4A3 v8::internal::Builtins::builtin_handle+319363
13: 00007FF705C79C8D v8::internal::SetupIsolateDelegate::SetupHeap+465453
14: 00007FF705C12092 v8::internal::SetupIsolateDelegate::SetupHeap+40498
15: 00007FF705C12092 v8::internal::SetupIsolateDelegate::SetupHeap+40498
16: 00007FF705C12092 v8::internal::SetupIsolateDelegate::SetupHeap+40498
17: 00007FF705CBBE1E v8::internal::SetupIsolateDelegate::SetupHeap+736190
18: 00007FF705C3233D v8::internal::SetupIsolateDelegate::SetupHeap+172253
19: 00007FF705C0FC3C v8::internal::SetupIsolateDelegate::SetupHeap+31196
20: 00007FF705ADFB7F v8::internal::Execution::CallWasm+1839
21: 00007FF705ADFC8B v8::internal::Execution::CallWasm+2107
22: 00007FF705AE06CA v8::internal::Execution::TryCall+378
23: 00007FF705AC0CE5 v8::internal::MicrotaskQueue::RunMicrotasks+501
24: 00007FF705AC0A40 v8::internal::MicrotaskQueue::PerformCheckpoint+32
25: 00007FF7053A0B40 node::CallbackScope::~CallbackScope+672
26: 00007FF7053A0F7B node::CallbackScope::~CallbackScope+1755
27: 00007FF7053A13C1 node::MakeCallback+209
28: 00007FFEC0FB5E05 
29: 00007FFEC0FB6AD6 
30: 00007FFEC0FB60D4 
31: 00007FFEC0FB63C2 
32: 00007FF7053D4C60 uv_timer_stop+560
33: 00007FF7053D4D37 uv_timer_stop+775
34: 00007FF7053D152B uv_async_send+331
35: 00007FF7053D0CBC uv_loop_init+1292
36: 00007FF7053D0E5A uv_run+202
37: 00007FF70525B04C v8::internal::wasm::SignatureMap::Freeze+35292
38: 00007FF705256B28 v8::internal::wasm::SignatureMap::Freeze+17592
39: 00007FF7053C165D uv_poll_stop+557
40: 00007FF7061888F0 v8::internal::compiler::RepresentationChanger::Uint32OverflowOperatorFor+283808
41: 00007FFECD7B7974 BaseThreadInitThunk+20
42: 00007FFED022A2F1 RtlUserThreadStart+33
npm ERR! code ELIFECYCLE
npm ERR! errno 3221225477
npm ERR! solid-simple-table@0.2.6 build: `cross-env NODE_ENV=production parcel build --target main --target types --target module src/SimpleTable.tsx && npm run style`
npm ERR! Exit status 3221225477
```

Ubuntu Error:
https://github.com/aminya/solid-simple-table/pull/11/checks?check_run_id=2816163837#step:4:576
```js

#
# Fatal error in , line 0
# Check failed: result.second.
#
#
#
#FailureMessage Object: 0x7f5a20bb0850
 1: 0xa8f951  [node]
 2: 0x1a0cd94 V8_Fatal(char const*, ...) [node]
 3: 0xe7a249 v8::internal::GlobalBackingStoreRegistry::Register(std::shared_ptr<v8::internal::BackingStore>) [node]
 4: 0xbc5088 v8::ArrayBuffer::GetBackingStore() [node]
 5: 0x9fa1af node::Buffer::Data(v8::Local<v8::Value>) [node]
 6: 0x7f5a281e4c7c  [/home/runner/work/solid-simple-table/solid-simple-table/node_modules/lmdb-store/build/Release/lmdb-store.node]
 7: 0x7f5a281e133c  [/home/runner/work/solid-simple-table/solid-simple-table/node_modules/lmdb-store/build/Release/lmdb-store.node]
 8: 0xc03d0b  [node]
 9: 0xc052b6  [node]
10: 0xc05936 v8::internal::Builtin_HandleApiCall(int, unsigned long*, v8::internal::Isolate*) [node]
11: 0x1423359  [node]
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! solid-simple-table@0.2.6 build: `cross-env NODE_ENV=production parcel build --target main --target types --target module src/SimpleTable.tsx && npm run style`
npm ERR! Exit status 1
npm ERR! 
```

MacOS error
https://github.com/aminya/solid-simple-table/pull/11/checks?check_run_id=2816163911#step:4:53
```js
#
# Fatal error in , line 0
# Check failed: result.second.
#
#
#
#FailureMessage Object: 0x700012fc6990
 1: 0x100120612 node::NodePlatform::GetStackTracePrinter()::$_3::__invoke() [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
 2: 0x101039292 V8_Fatal(char const*, ...) [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
 3: 0x1004e30d4 v8::internal::GlobalBackingStoreRegistry::Register(std::__1::shared_ptr<v8::internal::BackingStore>) [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
 4: 0x1002086c6 v8::ArrayBuffer::GetBackingStore() [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
 5: 0x10008cb1b node::Buffer::Data(v8::Local<v8::Value>) [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
 6: 0x104db83ff EnvWrap::batchWrite(Nan::FunctionCallbackInfo<v8::Value> const&) [/Users/runner/work/solid-simple-table/solid-simple-table/node_modules/lmdb-store/prebuilds/darwin-x64/node.abi83.node]
 7: 0x104db9853 Nan::imp::FunctionCallbackWrapper(v8::FunctionCallbackInfo<v8::Value> const&) [/Users/runner/work/solid-simple-table/solid-simple-table/node_modules/lmdb-store/prebuilds/darwin-x64/node.abi83.node]
 8: 0x100267568 v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
 9: 0x100266afc v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
10: 0x100266222 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
11: 0x100a894b9 Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_BuiltinExit [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
12: 0x100a222a2 Builtins_InterpreterEntryTrampoline [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
13: 0x100a222a2 Builtins_InterpreterEntryTrampoline [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
14: 0x100a222a2 Builtins_InterpreterEntryTrampoline [/Users/runner/hostedtoolcache/node/14.17.0/x64/bin/node]
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! solid-simple-table@0.2.6 build: `cross-env NODE_ENV=production parcel build --target main --target types --target module src/SimpleTable.tsx && npm run style`
npm ERR! Exit status 1
```

It worked fine on 662.

<!--- Provide a general summary of the issue here -->

## 💻 Code Sample
Everything is here:
https://github.com/aminya/solid-simple-table/pull/11

<!-- Please provide a code repository, gist, code snippet or sample files to reproduce the issue -->

## 🌍 Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-nightly-722
| Node             | 14
| npm/Yarn         | 
| Operating System | Win 10, Ubuntu, Mac

<!-- Love parcel? Please consider supporting our collective:
👉  https://opencollective.com/parcel/donate -->

--- a/packages/core/cache/package.json
+++ b/packages/core/cache/package.json
@@ -21,7 +21,7 @@
   "dependencies": {
     "@parcel/logger": "2.0.0-beta.3.1",
     "@parcel/utils": "2.0.0-beta.3.1",
-    "lmdb-store": "^1.5.2"
+    "lmdb-store": "^1.5.4"
   },
   "peerDependencies": {
     "@parcel/core": "^2.0.0-alpha.3.1"

--- a/packages/transformers/js/core/Cargo.toml
+++ b/packages/transformers/js/core/Cargo.toml
@@ -8,9 +8,9 @@ edition = "2018"
 crate-type = ["rlib"]
 
 [dependencies]
-swc_ecmascript = { version = "0.38.0", features = ["parser", "transforms", "module", "optimization", "react", "typescript", "utils", "visit", "codegen", "utils"] }
-swc_ecma_preset_env = "0.22.1"
-swc_common = { version = "0.10.19", features = ["tty-emitter", "sourcemap"] }
+swc_ecmascript = { version = "0.39.0", features = ["parser", "transforms", "module", "optimization", "react", "typescript", "utils", "visit", "codegen", "utils"] }
+swc_ecma_preset_env = "0.23.0"
+swc_common = { version = "0.10.20", features = ["tty-emitter", "sourcemap"] }
 swc_atoms = "0.2.6"
 indoc = "1.0.3"
 serde = "1.0.123"

